shell.nix

```diff
@@ -7,7 +7,7 @@ let
   iohk-ops = iohkpkgs.iohk-ops;
   justIo = pkgs.mkShell {
     name = "io";
-    buildInputs = with pkgs; [ iohk-ops terraform_0_11 iohkpkgs.nixops cabal-install ];
+    buildInputs = with pkgs; [ iohk-ops terraform_0_11 iohkpkgs.nixops ];
```

Comment:
@disassembler, is `cabal-install` broken?  It's useful to have it..

Reply:
as discussed, this can be added back in if needed in another PR, but cabal-install was causing a full rebuild of GHC that we couldn't wait a few hours for to deploy the deployer.